### PR TITLE
Cherry pick of configuration files and change log for release v1.7.9

### PR DESCRIPTION
**Pull request by couralex6**
Cherry pick of configuration files and change log for release v1.7.9

This PR brings the configuration files for CNI v1.7.9 to `config/1.7/`

PR #1285 modified the config files in all the release directories under `config/`. I kept those changes while cherry-picking.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+* Improvement - [Adds http timeout to aws sessions](https://github.com/aws/amazon-vpc-cni-k8s/pull/1370) (#1370 by @couralex6)
+* Improvement - [Switch calico to be deployed with the Tigera operator](https://github.com/aws/amazon-vpc-cni-k8s/pull/1297) (#1297 by @tmjd)
+* Improvement - [Update calico to v3.17.1](https://github.com/aws/amazon-vpc-cni-k8s/pull/1328) (#1328 by @lwr20)
+* Improvement - [update plugins to v0.9.0](https://github.com/aws/amazon-vpc-cni-k8s/pull/1362) (#1362 by @fr0stbyte)
+* Improvement - [update github.com/containernetworking/plugins to v0.9.0](https://github.com/aws/amazon-vpc-cni-k8s/pull/1350) (#1350 by @fr0stbyte)
+* Bug - [Fix regex match for getting primary interface](https://github.com/aws/amazon-vpc-cni-k8s/pull/1311) (#1311 by @Jayanthvn)
+* Bug - [Output to stderr when no log file path is passed](https://github.com/aws/amazon-vpc-cni-k8s/pull/1275) (#1275 by @couralex6)
+* Bug - [Fix deletion of hostVeth rule for pods using security group](https://github.com/aws/amazon-vpc-cni-k8s/pull/1376) (#1376 by @SaranBalaji90)
+
+
```
Comment (haouc) [inline CHANGELOG.md]: nit: can we remove this extra new line.

**Review by haouc — APPROVED** [review]
Looks good to me.

**Review by jayanthvn — CHANGES_REQUESTED** [review]
Let's not cherrypick calico.yaml from 1.7 to master. Master will have the latest changes and we cherry picked few changes from master to 1.7 branch already.

**Review by jayanthvn — APPROVED** [review]
You are porting to v1.7 config folder in master so should be fine. Thanks. LGTM :)

**Pull request merged**